js/src/components/google-account-card/authorize-google-account-card.scss

```
@@ -0,0 +1,6 @@
+.gla-authorize-google-account-card {
+	&__alter-text {
```

❔ "alter" isn't that specific and meaningful IMO. How about "error"? 🤔 

```suggestion
	&__error-text {
```